torchrl/modules/tensordict_module/rnn.py

```diff
@@ -782,8 +788,8 @@ def _lstm(
             )
 
         # we only need the first hidden state
-        _hidden0_in = hidden0_in[:, 0]
-        _hidden1_in = hidden1_in[:, 0]
+        _hidden0_in = hidden0_in[..., 0, :, :]
+        _hidden1_in = hidden1_in[..., 0, :, :]
```

Comment:
This change is interesting. I'm guessing the shape of `hiddenN_in` is `(batch, sequence, num_lstm_layers, hidden_size)`.

Were we before not using the first hidden state of a sequence, but the first hidden layer in the LSTM?